### ISSUE TITLE
Input spikes from NeuroPixel probes

### ISSUE THREAD

**Issue by mariakesa** [bamsumit/slayerPytorch#73]
Hello,

Thanks for an awesome package:-)

I have NeuroPixel data with time stamps from spiking neurons (an array of spike times and an array of neuron indices). I would like to read it into Slayer and do regression on behavior PC's. What is the format required for Slayer? Does it accept spike times or should I bin the spikes? What if I want to read data in from a np array instead of a bs1 file? 

Any tips and hints would be much appreciated!

All the very best,
Maria

**Comment by bamsumit** [comment]
Yes you can use `snn.io.event` class to represent your event. Documentation [here](https://bamsumit.github.io/slayerPytorch/build/html/spikeIO.html#slayerSNN.spikeFileIO.event)

You can also read directly from numpy format using `snn.io.readNpSpikes` if your data is in xypt table format. Documentation [here](https://bamsumit.github.io/slayerPytorch/build/html/spikeIO.html#slayerSNN.spikeFileIO.readNpSpikes).
